website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -28,8 +28,8 @@ kind: EC2NodeClass
 metadata:
   name: default
 spec:
-  # Required, resolves a default ami and userdata
-  amiFamily: AL2
+  # Required, configures the bootstrapping mode
```

Comment:
wdyt about being more specific here? 
```suggestion
  # Required, configures the userdata that Karpenter injects into the bootstrapping script
```

Reply:
Right now it still configures more than just UserData, but also the default BlockDeviceMappings. I could go either way on adding more depth here or just expanding later in the docs.